### DISCUSSION TITLE
Can you add automatic user grouping to HFS?

### DISCUSSION THREAD

**Discussion by Win-92** [rejetto/hfs#706] [Ideas]
I would like that HFS would automatically group new users to my users group, so i can control them all easily, and admins i would manually move from users to administrators, but its not a feature yet, so can you help me with this?

**Comment by rejetto** [comment]
are you adding users manually in the admin-panel, like this? 
<img width="104" alt="image" src="https://github.com/user-attachments/assets/cb706425-befb-4578-b378-5161d86f485f">
